activerecord/lib/active_record/associations/association.rb

```diff
@@ -31,6 +31,18 @@ def initialize(owner, reflection)
         reset_scope
       end
 
+      # Checks whether lazy eager load of the association is possible
+      # and tries to preload if it is so.
+      def reader; end
```

Comment:
Why do we need this method stub?

Looks like the comment above is for the `lazy_preload`